docs/modules/ROOT/pages/node_pattern.adoc

```diff
@@ -316,6 +316,30 @@ We can use the `#prime?` method directly in the expression:
 (int #prime?)
 ----
 
+== Arguments for predicate and external methods
```

Comment:
predicates and external methods? I'm confused by this terminology. :-)

Reply:
Right, I'll replace "external methods" with "funcall", which is what is used elsewhere (`predicate?`, `#funcall`)